crates/api_models/src/payouts.rs

```diff
@@ -184,6 +184,8 @@ pub struct PayoutCreateRequest {
     /// Customer's phone country code. _Deprecated: Use customer object instead._
     #[schema(deprecated, max_length = 255, example = "+1")]
     pub phone_country_code: Option<String>,
+
+    pub payout_method_id: Option<String>,
```

Comment:
Add description specifying what this field does, in both request and response.

This is used for generating OpenAPI specs

Reply:
done